pkgs/games/shticker-book-unwritten/default.nix

```
@@ -0,0 +1,58 @@
+{ alsaLib, buildFHSUserEnv, fetchFromGitHub, lib, libX11, libXext, libglvnd, openssl, pkg-config, rustPlatform, stdenvNoCC }:
```

Only the dependencies for the unwrapped version should be included. Use the `targetPkgs` function to bring in other runtime dependencies for the buildFHSUserEnv
```suggestion
{ , buildFHSUserEnv, fetchFromGitHub, lib, openssl, pkg-config, rustPlatform }:
```